plugins/lite3_db/Makefile

```diff
@@ -0,0 +1,24 @@
+
+PATH := $(PATH):/home/harvey/links/deriving
+
+SQLITE3_LIBDIR=$(shell ocamlfind query sqlite3)
+
+ifneq ($(SQLITE3_LIBDIR),)
+   DB_AUXLIBS += $(SQLITE3_LIBDIR)
+   DB_CLIBS   += sqlite3
+   DB_LIBS    += sqlite3
+endif
+
+OCAMLLDFLAGS = -shared
+
+LIBS    = $(DB_LIBS) 
+INCDIRS = $(DB_AUXLIBS) $(EXTRA_INCDIRS) /home/harvey/dynlinx/links
```

Comment:
Another absolute path, there are lots in these Makefiles.  We may need to think of a better way to set this up so that the appropriate Links interfaces are available when compiling the plugins.

Reply:
Maybe I should use LINK_PREFIX in the Makefile.config?

Reply:
Since the plugins are compiled independently, it needs to know where to find necessary .cmx files.

Reply:
The `LINKS_PREFIX` variable is only available during the compilation of Links. Rather, I think we would want a variable called something like `LINKS_LD_LIBRARY_PATH` (analogous to `OCAML_LD_LIBRARY_PATH`) which would follow the typical `PATH` conventions, i.e. list of directory names separated by colons.